CommunicationLayer/MessagingFramework/OtherMessage.cpp

```diff
@@ -0,0 +1,27 @@
+#include "OtherMessage.h"
+#include "MessageDecodingHelpers.h"
+#include "MessageDefines.h"
+
+using namespace MessageDecodingHelpers;
+
+namespace
+{
+    const int GPS = 1;
+}
+
+OtherMessage::OtherMessage(const QByteArray& messageData)
+    : messageData_(messageData)
+{
+}
+
+unsigned char OtherMessage::getGps() const
+{
+    return getUnsignedChar(messageData_, GPS);
+}
+
+QString OtherMessage::toString() const
+{
+    QString messageString;
+    messageString += QString::number(MessageDefines::Other) + ", ";
+    return messageString;
```

Comment:
do you want me to remove all implementations of it? Or keep the files implementing it and just kill any references to it in the main files?